game.c

```diff
@@ -252,7 +255,7 @@ void sounder()
       }else{
 	sound_nosound();
       };
-      delay_wait(1193180/80);
+      delay_wait(1193180/1000+livealiens*500);
```

Comment:
This formula somehow looks wrong to me. The original value is `14914`. With this new formula the start value is now `21193` which makes the game quite a bit slower and then it decreases to an insane `1693` when there is only one ship left.

I think game speed should start with the original value and then converge to something like 2x speed when there is only 1 ship left. Something like `1193180/(160-2*livealiens)` probably? With this approach, the `livealiens == 40` case, i.e. division by zero, should be safe, as `sounder()` is only called after `calculate()` as long as `gameover` is `false`. What do you think?